models.py

```diff
@@ -144,6 +144,10 @@
 ON_HOLD = 10
 NO_LONGER_PURSUING = 1000 # insure bottom of list
 
+RELEASE_IMPL_STATES = {
+  ENABLED_BY_DEFAULT, DEPRECATED, REMOVED, ORIGIN_TRIAL, INTERVENTION,
```

Comment:
Should `BEHIND_A_FLAG` be part of this list?

Reply:
I thought that you didn't want it to be considered released.  If I add BEHIND_A_FLAG here, then a feature that has that implementation status and also has a milestone set would show up in a milestone query.

Should I add it or not?

Reply:
I guess I've been a bit inconsistent about that. If we want engineers to experiment, then they need to be able to find out what they can experiment with. I think it should be added.

Reply:
ok will do